sdk/formrecognizer/Azure.AI.FormRecognizer/samples/README.md

```diff
@@ -19,6 +19,10 @@ Azure Cognitive Services Form Recognizer is a cloud service that uses machine le
 - Custom - Build custom models to extract text, field values, selection marks, and table data from documents. Custom models are trained with your own data, so they're tailored to your documents.
 
 ## Common scenarios samples for SDK
+- [Analyze the layout of a document](https://github.com/Azure/azure-sdk-for-net/tree/main/sdk/formrecognizer/Azure.AI.FormRecognizer/samples/Sample_AnalyzeLayout.md)
```

Comment:
should this be Extract instead of analyze? According to Paul's terminology, everything here is just extraction.
See https://github.com/Azure/azure-sdk-for-python/tree/main/sdk/formrecognizer/azure-ai-formrecognizer#azure-form-recognizer-client-library-for-python

I suggest changing the file name too